vendors/nordic/boards/nrf52840-dk/ports/ble/iot_ble_hal_common_gap.c

```diff
@@ -158,8 +158,8 @@ ble_gap_sec_params_t xSecurityParameters =
     .keypress       = 0,
     .io_caps        = BLE_GAP_IO_CAPS_DISPLAY_YESNO,
     .oob            = 0,
-    .min_key_size   = 7,
-    .max_key_size   = 16,
+    .min_key_size   = btKEY_MAX_LEN,
```

Comment:
Is it a typo? It should be set to 'btKEY_MIN_LEN'?

Reply:
It is not :), we increase the min size to max. But I changed it anyway for less confusion